<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ElecClass</class>
 <widget class="QMainWindow" name="ElecClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1686</width>
    <height>1097</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Elec</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QWidget" name="wgtTitle" native="true">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>150</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>150</height>
       </size>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="btnConfig">
         <property name="minimumSize">
          <size>
           <width>96</width>
           <height>96</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>96</width>
           <height>96</height>
          </size>
         </property>
         <property name="text">
          <string>系统信息</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnQuery">
         <property name="minimumSize">
          <size>
           <width>96</width>
           <height>96</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>96</width>
           <height>96</height>
          </size>
         </property>
         <property name="text">
          <string>查询</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnUser">
         <property name="minimumSize">
          <size>
           <width>96</width>
           <height>96</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>96</width>
           <height>96</height>
          </size>
         </property>
         <property name="text">
          <string>权限</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnQuit">
         <property name="minimumSize">
          <size>
           <width>96</width>
           <height>96</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>96</width>
           <height>96</height>
          </size>
         </property>
         <property name="text">
          <string>退出</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>1799</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="CenterWidget" name="wgtContent" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_2"/>
      <widget class="QWidget" name="page_4"/>
      <widget class="QWidget" name="page_5"/>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CenterWidget</class>
   <extends>QWidget</extends>
   <header location="global">CenterWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="Elec.qrc"/>
 </resources>
 <connections/>
</ui>
